.golangci.yml

```diff
@@ -30,3 +30,15 @@ issues:
   - linters:
     - dupl
     path: _test\.go
+  - linters:
```

Comment:
it's probably worth adding at least a one-line comment explaining what each of these rules are that are being excluded